docs/README.md

```diff
@@ -1,11 +1,20 @@
 # ideacrm
 
-VTEX Idea CRM first-party app
+The CRM Platform's omnichannel enables complete control of customer interactions with the enterprise, providing measurements and traceability. Its service channels are Chat, Phone, Email, SMS, Tickets, Social Networks and Applications.
```

Comment:
Idea CRM is a first party integration app for the platform's customer relationship management solution. It enables you to control customer interactions, providing measurements and trackability through Chat, Phone, Email, SMS, Tickets, Social Network and other applications. 

For more on the Idea CRM solution, please visit their [website](https://www.gvp.com.br/crm)